test-cloud/test-cloud-ha-cep/src/test/java/org/kie/cloud/hacep/HACEPEvaluateRuleIntegrationTest.java

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright 2019 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.kie.cloud.hacep;
+
+import java.io.IOException;
+import java.util.Properties;
+import java.util.concurrent.ThreadLocalRandom;
+
+import io.fabric8.kubernetes.api.model.ConfigMap;
+import org.assertj.core.api.Assertions;
+import org.junit.Test;
+import org.kie.cloud.api.DeploymentScenarioBuilderFactory;
+import org.kie.cloud.api.scenario.HACepScenario;
+import org.kie.cloud.openshift.resource.Project;
+import org.kie.cloud.openshift.resource.impl.ProjectImpl;
+import org.kie.cloud.tests.common.AbstractMethodIsolatedCloudIntegrationTest;
+import org.kie.hacep.sample.kjar.StockTickEvent;
+import org.kie.remote.CommonConfig;
+import org.kie.remote.RemoteStreamingKieSession;
+import org.kie.remote.TopicsConfig;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class HACEPEvaluateRuleIntegrationTest extends AbstractMethodIsolatedCloudIntegrationTest<HACepScenario> {
+
+    private static final Logger logger = LoggerFactory.getLogger(HACEPEvaluateRuleIntegrationTest.class);
+
+    @Override
+    protected HACepScenario createDeploymentScenario(DeploymentScenarioBuilderFactory deploymentScenarioFactory) {
+        return deploymentScenarioFactory.getHACepScenarioBuilder().build();
+    }
+
+    @Test
+    public void testEvaluateRule() throws InterruptedException {
+        final TopicsConfig topicsConfig = TopicsConfig.getDefaultTopicsConfig();
+        final Properties connectionProperties = deploymentScenario.getKafkaConnectionProperties();
+        connectionProperties.putAll(CommonConfig.getStatic());
+        logger.info("Waiting 120 s before starting of sending messages");
+        Thread.sleep(120000);
+        try (RemoteStreamingKieSession producer = RemoteStreamingKieSession.create(connectionProperties, topicsConfig)) {
+            for (int i = 0; i < 1; i++) {
+                StockTickEvent eventA = new StockTickEvent("RHT",
+                                                           ThreadLocalRandom.current().nextLong(80,
+                                                                                                100));
+                producer.insert(eventA);
+            }
+        } catch (IOException e) {
```

Comment:
You can directly throw an exception from test method, no need to wrap it.

Reply:
Fixed.